tests/test_python/test_python.py

```diff
@@ -0,0 +1,42 @@
+from textwrap import dedent
+
+from dxr.testing import SingleFileTestCase
+
+
+class PythonSingleFileTestCase(SingleFileTestCase):
+    """Test case suited to testing the Python plugin."""
+    enabled_plugins = ('pygmentize', 'python')
```

Comment:
It seems like the common pattern is to use lists for homogenous series like this and tuples for anonymous structs. What do you think?